client/src/usePaginatorControls.test.js

```diff
@@ -0,0 +1,121 @@
+            expect(setLabel).toHaveBeenCalledWith(currentPage, "testLabel");
+        });
+    });
+
+    describe("setLabelSuffix", () => {
+        it("sets the label suffix", async () => {
+            const replaceLabelPartSpy = jest.spyOn(MagicLabeler, "replaceLabelPart").mockReturnValue("testLabel");
+            getMagicLabel.mockReturnValue("testMagicLabel");
+            const { result } = await renderHook(() => usePaginatorControls(currentPage, getMagicLabel, setLabel));
+
+            act(() => {
+                result.current.action.setLabelSuffix("test");
+            });
+
+            expect(replaceLabelPartSpy).toHaveBeenCalledWith("testMagicLabel", "suffix", "test", true);
+            expect(setLabel).toHaveBeenCalledWith(currentPage, "testLabel");
+        });
+    });
+
+    describe("toggleCase", () => {
+        it("toggles the label case", async () => {
+            const toggleCaseSpy = jest.spyOn(MagicLabeler, "toggleCase").mockReturnValue("testLabel");
+            getMagicLabel.mockReturnValue("testMagicLabel");
+            const { result } = await renderHook(() => usePaginatorControls(currentPage, getMagicLabel, setLabel));
+
+            act(() => {
+                result.current.action.toggleCase("test");
+            });
+
+            expect(toggleCaseSpy).toHaveBeenCalledWith("testMagicLabel");
+            expect(setLabel).toHaveBeenCalledWith(currentPage, "testLabel");
+        });
+    });
+
+    describe("toggleRoman", () => {
+        it("toggles the label to roman numerals", async () => {
+            const toggleRomanSpy = jest.spyOn(MagicLabeler, "toggleRoman").mockReturnValue("testLabel");
+            getMagicLabel.mockReturnValue("testMagicLabel");
+            const { result } = await renderHook(() => usePaginatorControls(currentPage, getMagicLabel, setLabel));
+
+            act(() => {
+                result.current.action.toggleRoman("test");
+            });
+
+            expect(toggleRomanSpy).toHaveBeenCalledWith("testMagicLabel");
+            expect(setLabel).toHaveBeenCalledWith(currentPage, "testLabel");
+        });
+    });
+
+    describe("updateCurrentPageLabel", () => {
+        it("toggles the label to roman numerals", async () => {
+            const toggleRomanSpy = jest.spyOn(MagicLabeler, "toggleRoman").mockReturnValue("testLabel");
+            getMagicLabel.mockReturnValue("testMagicLabel");
+            const { result } = await renderHook(() => usePaginatorControls(currentPage, getMagicLabel, setLabel));
+
+            act(() => {
+                result.current.action.toggleRoman("test");
+            });
+
+            expect(toggleRomanSpy).toHaveBeenCalledWith("testMagicLabel");
+            expect(setLabel).toHaveBeenCalledWith(currentPage, "testLabel");
+        });
+    });
+
+    describe("updateCurrentPageLabel", () => {
+        it("toggles the label to roman numerals", async () => {
```

Comment:
The `it()` description here seems incorrect.

Reply:
I think this still needs to be revised -- maybe `it("sets the label of the current page")`

Reply:
Never mind, seeing it now. :-)